### PR TITLE
[13.0][IMP] web_refresher: Work as element of control panel

### PR DESCRIPTION
**Pull request by Tardo**
Now the button is not hidden if the pagination is not visible.

cc @tecnativa TT30229

### CODE REVIEW

**Review by cesar-tecnativa — APPROVED** [review]

**Comment by cesar-tecnativa** [comment]
@CarlosRoca13 please review

File: web_refresher/static/src/js/refresher.js [hunk 1]
```diff
@@ -1,25 +1,117 @@
-// Initial pager is located in source/addons/web/static/src/js/chrome/pager.js
-
-odoo.define("refresher.pager", function(require) {
+/* Copyright 2022 Tecnativa - Alexandre D. Díaz
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
```
Comment (pedrobaeza) [inline web_refresher/static/src/js/refresher.js]:
```suggestion
 * License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
```

File: web_refresher/static/src/xml/refresher.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2022 Tecnativa - Alexandre Díaz
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
```
Comment (pedrobaeza) [inline web_refresher/static/src/xml/refresher.xml]:
```suggestion
     License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I have pushed the license typo fix, so merging:

/ocabot merge major

**Comment by pedrobaeza** [comment]
Please fw-port it changes to v14 and v15

**Pull request merged**